.github/workflows/unittests.yml

```diff
@@ -29,6 +29,8 @@ jobs:
         run: tox -e spell
       - name: Run twine check
         run: tox -e twine
+      - name: Run build check
+        run: tox -e build
```

Comment:
FWIW I tested it without this patch and it failed as expected: https://github.com/evverx/scapy/actions/runs/4452762824/jobs/7820708250